components/media/PreviewMediaResolver.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <MediaResolver
+    :src="properSrc"
+    :mimeType="properType"
+    :poster="poster"
+    preview />
+</template>
+
+<script lang="ts" setup>
+import { computed, defineAsyncComponent, onMounted, ref } from '#app'
```

Comment:
do we really need this? I thought it would be without import

Reply:
edit: eslint throw ` is not defined` error without, might need some tweak in configuration

Reply:
same, I thought that should work without import because of this docs https://v3.nuxtjs.org/guide/concepts/auto-imports/

when I remove the `ref` and `computed`, the app can still render perfectly. no error in the terminal when building or development mode. the error only occurs in my editor

I am still unable to find the root cause, so I put them explicitly https://v3.nuxtjs.org/guide/concepts/auto-imports/

Reply:
maybe our tsconfig conflicting somewhere. I'm still not sure, need to test it in fresh nuxt2 and try nuxt-bridge

Reply:
Maybe it's comming from vue-eslint-parser config https://eslint.vuejs.org/user-guide/#faq, I had same problem with another project so I added explicit imports too

Reply:
thanks for the link, it helps. turns out the config conflict comes from `'eslint:recommended'` in `.eslintrc.js` and `".nuxt"` excluded in `tsconfig.json`